submodules/externals/srtp

```diff
@@ -1 +1 @@
-Subproject commit 4caa4843b4bfc54be596d309b5d00a81a35e3276
+Subproject commit 37448b2aebeb5e9bd2d39d3a82f0721123cdd4a2
```

Comment:
Looking at https://github.com/BelledonneCommunications/linphone-iphone, this is 4caa4843b4bfc54be596d309b5d00a81a35e3276, why would I want to use 37448b2aebeb5e9bd2d39d3a82f0721123cdd4a2?